skypy/gravitational_waves/tests/test_merger_rate.py

```diff
@@ -17,3 +21,52 @@ def test_abadie_rates():
     L_B_rate = b_band_merger_rate(L_10, population='NS-NS', optimism='low')
     table_value = abadie_table_III['NS-NS']['low']
     assert np.isclose(L_B_rate.to_value(1/units.year), table_value, rtol=1e-5)
+
+
+def test_artale_rates():
+
+    # Check the number of merger rates returned
+    redshifts = np.random.uniform(0., 3., 100)
+    stellar_masses = 10.**(9.0 + np.random.randn(100))
+    rates = m_star_merger_rate(redshifts, stellar_masses * units.Msun, population='NS-NS')
+    assert len(rates) == len(stellar_masses)
+
+    # Test that a luminosity of M_sol  returns a
```

Comment:
```suggestion
    # Test that a luminosity of M_sol returns a
```

I am confused. Is it solar luminosity or mass?